src/components/GameItem.js

```diff
@@ -4,26 +4,40 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
+  constructor() {
+    super();
+    this.state = {
+      className: "game-item"
+    };
+  }
+
   static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
   onItemClicked = () => {
-    // Fill this in for Wave 2!
+    // Fill this in for Wave2
+    if (this.props.type === 'litter'){
+      this.setState({className: "game-item spotted-litter"})
+      this.props.onItemClicked();
+
+    } else { 
```

Comment:
what would you need to change in order to keep a player from scoring more than once on the same piece of litter.